provider/cluster_rosa_classic_state.go

```diff
@@ -85,3 +86,11 @@ type AdminCredentials struct {
 	Username types.String `tfsdk:"username"`
 	Password types.String `tfsdk:"password"`
 }
+
+type DefaultIngress struct {
+	RouteSelectors           types.Map    `tfsdk:"route_selectors"`
```

Comment:
Not sure how it is going to work with TF, but there is also 2 extra attributes only for day 2 operations. Which are the components route hostname and components route tls secret ref

Reply:
Wanted to start with easy ones, will do it as second pr

Reply:
It will be following PR